Adapter.C

```diff
@@ -10,7 +10,7 @@ preciceAdapter::Adapter::Adapter(const Time& runTime, const fvMesh& mesh)
 : runTime_(runTime),
   mesh_(mesh)
 {
-    adapterInfo("Loaded the OpenFOAM-preCICE adapter - unreleased version.", "info");
+    adapterInfo("Loaded the OpenFOAM-preCICE adapter - v1.0.1.", "info");
```

Comment:
Discussion between @uekerman, @DavidSCN, and me:

- [Versioning strategy](https://github.com/precice/openfoam-adapter/issues/52) writes:
   - > Minor: new features (nearest-projection mapping, fluid-fluid coupling, ...). New tutorials may need this new version.
   - > Bugfix / revision: small nice-to-have updates or bugfixes. The same tutorials run with any revision.
   - With this in mind, bugfix is ok.
 - The additional location checks is an important safety feature we should emphasize -> could go to a minor release.
 - In preCICE, we really use "bugfix" as "this is fixing something that was previously wrong and needs to be released immediately". This typically results in a cherry-pick. With a bugfix release, we want to be 100% sure that we don't introduce new bugs -> because we had updates for OpenFOAM v2112 (that may be breaking for some older releases or have other side effects), let's to minor.